project_issue_code/__openerp__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Michael Viriyananda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Project Issue Code",
+    "version": "9.0.1.0.0",
+    "summary": "Adding Field Code For Project Issue",
+    "author": "OpenSynergy Indonesia,Odoo Community Association (OCA)",
```

Comment:
The author is OpenSynergy and but the code is copyright from Michael Viriyananda instead of OpenSynergy?
Can you explain please?